pandas/tseries/index.py

```diff
@@ -694,6 +694,21 @@ def astype(self, dtype):
         else:  # pragma: no cover
             raise ValueError('Cannot cast DatetimeIndex to dtype %s' % dtype)
 
+    def strftime(self, date_format):
```

Comment:
hmm, not to make this more complicated, but maybe move to ``tseries/base/DatelikeOps`` (new mixin), that both ``DatetimeIndex/PeriodIndex``, but NOT ``TimedeltaIndex`` inherit from.